networks/p2p/discover/node_test.go

```diff
@@ -53,13 +53,13 @@ func ExampleNewNode() {
 	id := MustHexID("1dd9d65c4552b5eb43d5ad55a2ee3f56c6cbc1c64a5c8d659f51fcd51bace24351232b8d7821617d2b29b54b81cdefb9b3e9c37d7fd5f63270bcc9e1a6f6a439")
 
 	// Complete nodes contain UDP and TCP endpoints:
-	n1 := NewNode(id, net.ParseIP("2001:db8:3c4d:15::abcd:ef12"), 52150, 30303, NodeTypeUnknown)
+	n1 := NewNode(id, net.ParseIP("2001:db8:3c4d:15::abcd:ef12"), 52150, 30303, nil, NodeTypeUnknown)
 	fmt.Println("n1:", n1)
 	fmt.Println("n1.Incomplete() ->", n1.Incomplete())
 
 	// An incomplete node can be created by passing zero values
 	// for all parameters except id.
-	n2 := NewNode(id, nil, 0, 0, NodeTypeUnknown)
+	n2 := NewNode(id, nil, 0, 0, nil, NodeTypeUnknown)
```

Comment:
@winnie-byun Could you add raw url test cases with subport?

Reply:
review reflected :)